src/psyclone/tests/psyir/transformations/transformations_test.py

```diff
@@ -583,6 +587,141 @@ def test_omploop_trans_new_options(sample_psyir):
                 in str(excinfo.value))
 
 
+def test_omplooptrans_apply_nowait(fortran_reader, fortran_writer):
```

Comment:
Any idea why this is showing as a new test or is it something you really have just added?

Reply:
I think I had a merge conflict in this file so I suspect its just due to that.